admin-portal/src/pages/needUpload/needUpload.tsx

```diff
@@ -1,25 +1,69 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router";
 import { PageSelection } from "../../types/pages";
 import { Page } from "layout/page";
 import "./needUpload.css";
 import { AidPackageService } from "apis/services/AidPackageService";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 export function NeedUpload() {
-  const [file, setFile] = useState();
+  const [file, setFile] = useState(null);
+  const [errorList, setErrorList] = useState([]);
+  const navigate = useNavigate();
 
   function handleChange(event: any) {
+    setErrorList([]);
     setFile(event.target.files[0]);
   }
 
   const handleSubmit = async (event: any) => {
     event.preventDefault();
+
     const formData = new FormData();
+
     if (file != undefined) {
       formData.append("file", file);
-      const data = await AidPackageService.postNeeds(formData);
-      console.log(data);
+      AidPackageService.postNeeds(formData)
```

Comment:
Let's use `async/await` instead of `then` to maintain the consistency of the codebase.